api/profile/profile_test.go

```diff
@@ -86,3 +84,21 @@ func TestProfileBasics(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, *p, *clone)
 }
+
+func TestAppPath(t *testing.T) {
+	t.Parallel()
+
+	dir := t.TempDir()
+
+	p := &profile.Profile{
+		WebProxyAddr:          "proxy:3088",
+		SSHProxyAddr:          "proxy:3023",
+		Username:              "testuser",
+		ForwardedPorts:        []string{},
+		DynamicForwardedPorts: []string{},
```

Comment:
These can probably be omitted (unless you want to make it explicit these are empty for some reason).

Reply:
Done